Sources/Impl/PinLayoutImpl+Warning.swift

```diff
@@ -20,6 +20,8 @@
 #if os(iOS) || os(tvOS)
 import UIKit
 
+fileprivate var numberFormatter: NumberFormatter?
```

Comment:
Using a lazy variable would make more sense in my opinion and have the exact same effect than the var + func technique. Also, it would have the effect of eliminating the need for an optional.

```swift
fileprivate var numberFormatter: NumberFormatter = {
    let numberFormatter = NumberFormatter()
    numberFormatter.numberStyle = .decimal
    numberFormatter.decimalSeparator = "."
    return numberFormatter
}()
```

Reply:
Thanks, way better this way 🙂